docs/operations/etcd/cluster-migration-node-to-pod.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Etcd cluster migration"
+---
+
+
+This procedure explains how to migrate data from an etcd cluster running
+outside a Kubernetes cluster towards an installation using the Ondat etcd
+Operator. As a result the Ondat cluster will use the etcd running as Pods in
+Kubernetes.
+
+## Prerequisites
+
+- Kubectl
+- Helm
+
+It is assumed that both etcd clusters in this procedure are using mTLS.
+
+## Procedure
+
+### Option A - Manual process
+
+1. Backup the TLS artifacts from the current etcd cluster (if current etcd uses mTLS)
+    The Secret with the TLS material is usually named `storageos-etcd-secret`
+    or `etcd-client-tls`
+
+    ```bash
+    $ kubectl get secret \
+        -n storageos \
+        -o yaml \
+        etcd-client-tls > etcd-storageos-tls-secret-backup.yaml
+    ```
+
+1. Deploy etcd cluster in Kubernetes
+
+    ⚠️  It is required to select a storageClass other than ondat/storageos to
+    run etcd in the cluster. If there is none available in the cluster, you can
+    run the following to deploy a [Local Path
```

Comment:
We don't support running "Local Path Provisioner" in production.

We should mention that here